swift/ios-shoppe-demo/Views/CartQuantityTableViewCell.swift

```diff
@@ -30,7 +30,7 @@ class CartQuantityTableViewCell: UITableViewCell {
 
         titleLabel.text = product.title
         quantityLabel.text = "x\(product.quantity)"
-        totalLabel.text = "$\(product.getTotal())"
+        totalLabel.text = product.getTotal().convertDoubleToCurrency()
```

Comment:
Yep, I'd say this is the View and to keep the addition of currency in it and not the Model.